stack.yaml

```diff
@@ -8,18 +8,22 @@ nix:
   shell-file: shell.nix
 
 extra-deps:
+- aeson-options-0.0.0
+- base58-bytestring-0.1.0
 - log-warper-1.8.11
 - pretty-tree-0.1.0.0
 - time-units-1.0.0
 - componentm-0.0.0.2
 - rio-0.1.2.0
 - serokell-util-0.8.0
+- formatting-6.2.5
 - typed-process-0.2.2.0
 - teardown-0.5.0.0
 - unliftio-0.2.7.0
+- yaml-0.8.32
```

Comment:
`yaml-0.8.31.1` from the lts is not good enough?

Reply:
Yes, it's not: https://serokell.slack.com/archives/C742ZKYF5/p1531389945000182